ReflectometryServer/config_helper.py

```diff
@@ -213,7 +227,7 @@ def add_mode(name, is_disabled=False):
     return name
 
 
-def add_driver(driver, marker=None):
+def add_driver(driver: IocDriver, marker: Union[int, None] = None) -> IocDriver:
```

Comment:
Just fyi (not important for this PR), you can use `int | None` for this type hint, and generally `TypeA | TypeB | TypeC` where previously `Union[TypeA, TypeB, TypeC]` might have been used.

Reply:
ah cheers. i did that originally (after switching from typescript...!) and thought it was weird the IDE wasn't moaning at me.